nixos/modules/services/backup/restic.nix

```diff
@@ -96,7 +96,7 @@ in
           type = types.listOf types.str;
           default = [];
           description = ''
-            Which paths to backup.
+            Which paths to backup.  May be left empty if pruneOpts is non-empty, in order to create a prune-only job.
```

Comment:
It'd be better if it was `null`, instead of an empty string, IMHO. I think that better communicates that there will be no command run.

`types.listOf (types.nullOr types.str)`

Reply:
Shouldn't it be `types.nullOr (types.listOf type.str)` then?

Reply:
Yes, sorry.

Reply:
Done.